File: ProjectEuler/009/009.py
import math

def is_triplet(a, b, c):
	if(a <= b and b <= c):
		return False
	pyth = math.sqrt(a**2 + b**2)
	return c == pyth

for a in range(1,1000):
	for b in range(1,1000):
		for c in range(1,1000):
			if(a + b + c == 1000 and is_triplet(a, b, c)):
				print(a*b*c)
				break;

File: ProjectEuler/009/009.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool is_triplet(int a, int b, int c){
	if(a >= b || b >= c) return false;

	double pyth = sqrt(pow(a, 2) + pow(b, 2));
	return c == pyth;
}

int main(){
	for(int a = 1; a < 1000; a++){
	for(int b = 1; b < 1000; b++){
	for(int c = 1; c < 1000; c++){
		if(a + b + c == 1000 && is_triplet(a, b, c)){
			cout << a*b*c;
		}
	}
	}
	}
	return 0;
}
